libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -164,20 +164,25 @@ class AP_Motors {
     // set loop rate. Used to support loop rate as a parameter
     void                set_loop_rate(uint16_t loop_rate) { _loop_rate = loop_rate; }
 
-    enum pwm_type { PWM_TYPE_NORMAL=0, PWM_TYPE_ONESHOT=1, PWM_TYPE_ONESHOT125=2, PWM_TYPE_BRUSHED16kHz=3 };
+    enum pwm_type {
+        PWM_TYPE_NORMAL = 0,
```

Comment:
I like to have no column for constant specification. Please tell me about the judgment of not adjusting the column.